pageserver/src/basebackup.rs

```diff
@@ -320,6 +320,8 @@ where
                     ));
                     info!("Replication slot {} restart LSN={}", path, restart_lsn);
                     min_restart_lsn = Lsn::min(min_restart_lsn, restart_lsn);
+                } else if path == "pg_logical/replorigin_checkpoint" {
+                    continue; // skip repliction origin fiel because we will generate new one
```

Comment:
This is just for backwards-compatibility with tenants that already started using logical replication, right? Let's mention it in the comment.

Reply:
Yes, I am going to remove storing `pg_logical/replorigin_checkpoint` in future.
But as you have noticed, we need it for backward compatibility.
Also I do not want to include in this PR changes in Postgres code.

I will update comment.

Reply:
I have added more comments and add tests (it reproduced the problem on unpatched Neon).